threescale_api/resources.py

```diff
@@ -315,6 +321,19 @@ def __init__(self, *args, entity_name='application', entity_collection='applicat
     def url(self) -> str:
         return self.parent.url + '/keys'
 
+    def delete(self, key, **kwargs) -> bool:
+        """Delete resource
+        Args:
+            entity_id(int): Entity id
+            **kwargs: Optional args
+
+        Returns(bool): True if the resource has been successfully deleted
+        """
+        log.info(self._log_message("[DELETE] Delete ", entity_id=key, args=kwargs))
+        url = self._entity_url(entity_id=key)
+        response = self.rest.delete(url=url, **kwargs)
+        return response.ok
+
```

Comment:
Once again, why introduce this method?

The `DefaultClient` contains delete method.
https://github.com/pestanko/3scale-api-python/blob/master/threescale_api/defaults.py#L89

You copied it from the default client...